shared/output/cloud/language_v1/lib/google/cloud/language/v1/language_service/client.rb

```diff
@@ -59,57 +59,57 @@ def self.configure
                 parent_config = while namespace.any?
                                   parent_name = namespace.join "::"
                                   parent_const = const_get parent_name
-                                  break parent_const.configure if parent_const&.respond_to? :configure
+                                  break parent_const.configure if parent_const.respond_to? :configure
                                   namespace.pop
                                 end
                 default_config = Client::Configuration.new parent_config
 
                 default_config.rpcs.analyze_sentiment.timeout = 600.0
                 default_config.rpcs.analyze_sentiment.retry_policy = {
                   initial_delay: 0.1,
-                  max_delay:     60.0,
-                  multiplier:    1.3,
-                  retry_codes:   [4, 14]
+              max_delay: 60.0,
```

Comment:
this can't be right

Reply:
This one's a genuine puzzler. I have no idea why Rubocop is "fixing" the indentation in this way, nor why it _allows_ this indentation when it subsequently runs during tests. I'm inclined to say "meh", this is generated code. Since the change doesn't hurt anything, maybe we don't worry about it.

Reply:
we can look into it later

Reply:
will do